src/elements/FlexRow/FlexRow.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import { getBreakpoint, numberLikeProp } from '../../util/helpers';
+import Common from '../Common/Common';
+
+const FlexRow = ({ children, breakMedium, breakSmall, breakCustom, gutter = 'none', ...other }) => {
+	const combinedClasses = classNames(
+			'ui-flex-row',
+			`gutter-${gutter}`,
```

Comment:
If `none` is the default, should we instead just make this conditional and only add a class if `gutter` is present? Also do we maybe want to allow different gutter's at different resolutions (gutterSmall, gutterMedium, etc) like we do on the Grid?

Reply:
I think we need to revisit breakpoints as a general class and provide a simpler opportunity to adjust these at the app level via media queries